### ISSUE TITLE
"http" directive is duplicate

### ISSUE THREAD

**Issue by domchristie** [basecamp/powprox#7]
Thanks for this!

I am getting the following when attempting to run `bin/powprox`:

```
nginx: [emerg] "http" directive is duplicate in /usr/local/etc/nginx/conf.d/powprox.conf:4
nginx: configuration file /usr/local/etc/nginx/nginx.conf test failed
```

Any ideas what I might be able to do to fix?

Thanks again.

**Comment by mmlindeboom** [comment]
I had this error. I just opened `/usr/local/etc/nginx/nginx.conf` and removed the entire http directive. Worked fine after that! This fix obviously depends on your use of nginx for anything else locally.

**Comment by jeremy** [comment]
Fixed by 230c699d63ac2d384338c8dc9c85509945148be0
